easybuild/easyconfigs/c/CubeGUI/CubeGUI-4.6-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,59 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2019 Juelich Supercomputing Centre, Germany
+# Authors::   Markus Geimer <m.geimer@fz-juelich.de>
+# License::   3-clause BSD
+#
+# This work is based on experiences from the UNITE project
+# http://apps.fz-juelich.de/unite/
+##
+
+easyblock = 'EB_Score_minus_P'
+
+name = 'CubeGUI'
+version = '4.6'
+
+homepage = 'https://www.scalasca.org/software/cube-4.x/download.html'
+description = """
+ Cube, which is used as performance report explorer for Scalasca and Score-P,
+ is a generic tool for displaying a multi-dimensional performance space
+ consisting of the dimensions (i) performance metric, (ii) call path, and
+ (iii) system resource. Each dimension can be represented as a tree, where
+ non-leaf nodes of the tree can be collapsed or expanded to achieve the
+ desired level of granularity.
+
+ This module provides the Cube graphical report explorer.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+source_urls = ['https://apps.fz-juelich.de/scalasca/releases/cube/%(version_major_minor)s/dist']
+sources = [SOURCELOWER_TAR_GZ]
+patches = ['CubeGUI-4.6_qtlibs_symlink_realpath.patch']
```

Comment:
Adding this patch breaks the build for me. See https://gist.github.com/branfosj/2aa7d6577f8d530e022224b22636ec5b#file-cubegui-4-6-gcccore-10-3-0_partial-log-L475

Reply:
The patch breaks for me on another system. This is because a `*.m4` file was modified in the build-config and `autoreconf` has to executed. I have added that as a preconfig step in the latest commit. Can you check if it breaks again?